lib/tapioca/generator.rb

```diff
@@ -108,6 +108,52 @@ def build_todos
       say("Please review changes and commit them.", [:green, :bold])
     end
 
+    sig { params(requested_constants: String).void }
+    def build_dsl(*requested_constants)
+      base_directory = Pathname.new("#{Config::DEFAULT_RBIDIR}/dsl")
+      generation_command_template = config.generate_command
+
+      loader = Loader.new(Gemfile.new)
```

Comment:
I wonder if this could be a pipeline "step" that would be required by all other generators for DSL.